docs/source/community/host-guide.md

```diff
@@ -34,66 +34,66 @@ Each meeting should have a list of topics that are generally discussed. Contribu
 
 ## &#x1F4AA; Host Responsibilities
 
-First and foremost, the host should facilitate the flow of a meeting. That means:
+The host should facilitate the flow of a meeting. That means:
 
+- Soliciting topics for the agenda:
+  - *Share the agenda document in the chat multiple times. On Zoom, for example, new participants do not see previous chat messages.*
 - Going through the agenda:
   - *Do time checks/pause + move on to hit all items when there are time concerns*
 - Guiding discussion:
   - *Read chats aloud for participants + the recording*
 - Maintaining order / Making sure everyone has a chance to speak:
   - *Interject when someone has their hand raised*
-- Mention/link to the [Jupyter Code of Conduct](https://jupyter.org/governance/conduct/code_of_conduct.html) and how everyone is bound to it (including the host)
+- Mention/link to the [Jupyter Code of Conduct](https://jupyter.org/conduct) and how everyone is bound to it (including the host)
```

Comment:
The original link works?

Reply:
It does, but especially if we're asking people to mention the link out loud, the shorter URL might be more memorable.